7081: Update Copyright year to 2021, Validate and update license attributes

guruhb opened the pull request:
Updated Copyright year to 2021 for below 
1. License show in "About" dialog
2. Copyright year in License file 
3. Update site landing page 
4. added a helper script which can be used to update copyright year for modified git files.

<!-- Anything below this marker will be automatically updated, please do not edit manually! -->
---------
### Progress
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JMC-7081](https://bugs.openjdk.java.net/browse/JMC-7081): Update Copyright year to 2021, Validate and update license attributes


### Reviewers
 * [Marcus Hirt](https://openjdk.java.net/census#hirt) (@thegreystone - **Reviewer**)


### Download
`$ git fetch https://git.openjdk.java.net/jmc pull/190/head:pull/190`
`$ git checkout pull/190`

thegreystone commented:
Pinged @guruhb about not hardcoding the from year in the copyright updating script - he'll said he'll fix before pushing.

guruhb commented:
> Pinged @guruhb about not hardcoding the from year in the copyright updating script - he'll said he'll fix before pushing.

Modified the script + also fixed to work with bash and Non Mac terminal.

guruhb commented:
/integrate
